src/disc/stk/Albany_STKDiscretization.cpp

```diff
@@ -804,6 +792,51 @@ STKDiscretization::writeSolutionMVToFile(
 #endif
 }
 
+void STKDiscretization::addSolutionField(const std::string & fieldName,const std::string & blockId)
+{
+#if 0
+   TEUCHOS_TEST_FOR_EXCEPTION(!validBlockId(blockId),ElementBlockException,
+                      "Unknown element block \"" << blockId << "\"");
+   std::pair<std::string,std::string> key = std::make_pair(fieldName,blockId);
+
+   // add & declare field if not already added...currently assuming linears
+   if(fieldNameToSolution_.find(key)==fieldNameToSolution_.end()) {
+      SolutionFieldType * field = metaData_->get_field<SolutionFieldType>(stk::topology::NODE_RANK, fieldName);
+      if(field==0)
+         field = &metaData_->declare_field<SolutionFieldType>(stk::topology::NODE_RANK, fieldName);
+      if ( initialized_ )  {
+        metaData_->enable_late_fields();
+        stk::mesh::FieldTraits<SolutionFieldType>::data_type* init_sol = nullptr;
+        stk::mesh::put_field_on_mesh(*field, metaData_->universal_part(),init_sol );
+      }
+      fieldNameToSolution_[key] = field;
+   }
+#endif
+}
+
+void STKDiscretization::addCellField(const std::string & fieldName,const std::string & blockId)
+{
+#if 0
```

Comment:
Is the code in the `#if 0` supposed to be used later? 

Also, who uses the `addCellField` and `addSolutionField` methods?

Reply:
Nvm, I found it in the STKConnManager.

Reply:
👍